splinter/driver/lxmldriver.py

```diff
@@ -77,8 +81,8 @@ def serialize(self, form):
     def submit(self, form):
         method = form.attrib.get("method", "get").lower()
         action = form.attrib.get("action", "")
-        if action.strip() != ".":
-            url = os.path.join(self._url, action)
+        if action.strip() not in (".", ""):
+            url = urllib.parse.urljoin(self._url, action)
```

Comment:
How the urljoin is imported, we should use it directly. This will fix the tests that are broken:

```suggestion
            url = urljoin(self._url, action)
```